src_testbed/testbed.rs

```diff
@@ -1087,7 +1087,7 @@ impl State for Testbed {
         if true {
             let counters = self.mechanical_world.counters;
 
-            let mut profile = format!(
+            let profile = format!(
```

Comment:
We can't remove the warning because if we compile with the `fluids` feature, the `profile` variable is mutated [bellow](https://github.com/rustsim/nphysics/pull/239/files#diff-3c4febc5de7e3620423edc937ef3487aL1125). I think we should disable the warning instead:

```suggestion
            #[allow(unused_mut)]
            let mut profile = format!(
```

Reply:
Looks good to me. However this commit passed CI, which indicates that not all features are built in CI. Is it worth adding `cargo build --all-features` to the circle build workflow?

Reply:
Yeah, adding more build jobs would be useful. Let's keep this for another PR though.